### ISSUE TITLE
dutch translation broken

### ISSUE THREAD

**Issue by jjurkus** [nilsteampassnet/TeamPass#1150]
In the file includes/language/dutch.php there are a few things missing:

line 189 was:
'admin_path_to_upload_folder_tip' => '<span style=\'font-size:11px;max-width:300px;\',
This should be changed to:
'admin_path_to_upload_folder_tip' => '<span style=\'font-size:11px;max-width:300px;\'',

I have made a few changes to the dutch translation, because it was rather rough around the edges. I'm a native dutch speaker, and aware of most new spelling rules.
[dutch.txt](https://github.com/nilsteampassnet/TeamPass/files/82873/dutch.txt)

**Comment by jjurkus** [comment]
Oh, without the double ' in the dutch.php file it will give a blank screen in the browser for the user that has selected dutch language.
From /var/log/httpd/error_log:
[Fri Jan 08 15:51:19.922405 2016] [:error] [pid 14378] [client 10.23.33.69:52433] PHP Parse error:  syntax error, unexpected 'admin_path_to_upload_folder_ti' (T_STRING), expecting ')' in /opt/TeamPass-2.1.24.4/includes/language/dutch.php on line 189

With the included dutch.txt (renamed to dutch.php of course) this has been fixed.

**Comment by nilsteampassnet** [comment]
Thanks for this.

Can you please update in [DUTCH page](https://poeditor.com/projects/po_edit?id_language=41&id=16418) under Poeditor?

**Comment by jjurkus** [comment]
Ok, I will.

Also note that another person had fixed it in a pull-request, which I did not look at before posting this:
https://github.com/nilsteampassnet/TeamPass/pull/1089

(BTW, where is the forum? TeamPass tells me to post on the forum first, before making a new issue, but I'm kinda lost)

**Comment by jjurkus** [comment]
Well, I think I found the forum, but with only two messages in a language I can't read this doesn't seem to be very helpful.

Anyway, I have fixed a few dutch and english errors.
There are a few more problems, such as a concatenation of two strings which should form one word in dutch:
User and Account are translated seperately, however when used together it should not be 'gebruiker' + 'account', but 'gebruikersaccount'.
This is something minor, I will open an issue when I feel the rest of the translation isn't as kludgey as it is now.
